src/ports/status.ts

```diff
@@ -0,0 +1,118 @@
+import { IBaseComponent } from '@well-known-components/interfaces'
+import { BaseComponents } from '../types'
+
+export type ServiceStatus = {
+  time: number
+  version: string
+  commitHash: string
+}
+
+export type LighthouseStatus = ServiceStatus & {
+  realmName: string
+}
+
+export type IStatusComponent = IBaseComponent & {
+  getLambdasStatus(): Promise<ServiceStatus | undefined>
+  getContentStatus(): Promise<ServiceStatus | undefined>
+  getLighthouseStatus(): Promise<LighthouseStatus | undefined>
+}
+
+const STATUS_EXPIRATION_TIME_MS = 1000 * 60 * 5
```

Comment:
can we add a comment here that says `# 5mins`?

Reply:
of course!